### ISSUE TITLE
Commands queue

### ISSUE THREAD

**Issue by chelminski** [titivermeesch/CommandTimer#244]
Hi
Please add a command delay option so that commands are queued and executed at different intervals. For example, I have a problem with the server hanging when many worlds are generated at the same time. A command queue would have avoided the problems.
For example, the server issues the commands:
/cmt mw create world1
/cmt mw create world2
/cmt mw create world3
/cmt mw create world4
/cmt mw create world5
....

And your plugin would execute these commands sequentially, but not immediately. Only every e.g. 10 seconds.
/cmt mw create world1 - now
/cmt mw create world2 -after 10s
/cmt mw create world3 -after 20s
/cmt mw create world4 -after 30s
/cmt mw create world5 -after 40s

Alternatively, if your plugin already has this function, please write how. I couldn't find any information on the plugin's website :(

**Comment by titivermeesch** [comment]
This feature already exists and is documented here: https://titivermeesch.github.io/CommandTimer/docs/configuration/commands#individual-command-delays

There is also a ticket open to modify the existing behaviour but not super relevant: #243 243

**Comment by chelminski** [comment]
From what I can see, I can configure it to send specifically defined commands. What I want is more so that I can add one-time commands to the queue. I don't know what commands will be executed. It depends on the users. Maybe several users will want to add new worlds and delete worlds at the same time. Each command will look different (world name, remove/add). I am looking for a solution that will delay the execution of commands entered in the console with the prefix of your plugin. Would this be possible to implement?

**Comment by titivermeesch** [comment]
Ah I see. So it's more about executing commands with delays without having to configure a task then?

**Comment by chelminski** [comment]
Yes exactly. This is exactly what I meant, but I couldn't explain it simply 😅

**Comment by titivermeesch** [comment]
I'll take this as a feature request. I cannot promise any timeline right now because there is so much else that also needs to be done

**Comment by titivermeesch** [comment]
There is finally progress on this, available on branch `feat/execution-engine-v2`.
I'm hoping to release this still this month but it will require extensive testing:

<img width="795" height="76" alt="Image" src="https://github.com/user-attachments/assets/012de6e8-633d-4dd2-9b5d-20100299d4e9" />
